src/views/Home.vue

```diff
@@ -1,16 +1,73 @@
 <template id="home">
   <aws-authenticator />
+  <h1 class="text-center">Books</h1>
+  <el-table :data="books" stipe style="width: 100%">
+    <el-table-column
+      header-align="center"
+      align="center"
+      prop="title"
+      label="Title"
+    >
+    </el-table-column>
+    <el-table-column
+      header-align="center"
+      align="center"
+      prop="author"
+      label="Author"
+    >
+    </el-table-column>
+    <el-table-column
+      header-align="center"
+      align="center"
+      prop="publicationDate"
+      label="Publication Date"
+    >
+    </el-table-column>
+  </el-table>
 </template>
 
 <script lang="ts">
 // vue
-import { defineComponent } from "vue";
+import { computed, defineComponent, inject, onMounted } from "vue";
 
 // components
 import AwsAuthenticator from "@/components/AwsAuthenticator.vue";
+import { useStore } from "vuex";
 
 export default defineComponent({
   name: "Home",
   components: { AwsAuthenticator },
+  setup() {
+    const store = useStore();
+    const axios: any = inject("axios");
+
+    // call to api to get books on created
+    axios
+      .get("http://localhost:3000/")
```

Comment:
put this in an environment variable and then load it like this 

```suggestion
      .get(process.env.API_URL)
```